package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "devDependencies": {
+    "eslint": "^8.23.1",
+    "eslint-config-kyle": "^9.3.0",
```

Comment:
Just checking but should this have your name in it?

Reply:
I've re-used a preset to save time, will remove it once #75 is fixed